Source/bmalloc/libpas/src/libpas/pas_probabilistic_guard_malloc_allocator.c

```diff
@@ -43,8 +43,13 @@
 #include "pas_random.h"
 #include <stdint.h>
 
+/* Total 10 {0 - 9} PGM entries are allowed for which metadata is kept alive */
```

Comment:
Curious how you reached the number 10. Is this from benchmarking the max amount before we see memory regressions?

Reply:
This is just to experiment as how the perf changes. This can be changed based on the Perf results.

Reply:
I think we should definitely try increasing this? My understanding is this should probably be as high as possible so long as we don't regress performance, so we should figure out approximately what that limit is via some testing.

Reply:
Will run Perf tests before increasing this number.